netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -655,21 +640,20 @@ const getStoredData = () => {
   return { storedData, storedTimeStamp };
 };
 
+const ONE_DAY_IN_MS = 24 * 60 * 60 * 1000;
```

Comment:
thanks @OchiengPaul442 , please share more insights about what this does?

Reply:
@Baalmart, the code just calculates the 24hr time in milliseconds. It's then used to update the data local storage only when 24hr have elapsed.